js/src/run_trees.ts

```diff
@@ -62,7 +62,7 @@ export class RunTree implements BaseRun {
       api_key: getEnvironmentVariable("LANGCHAIN_API_KEY"),
       caller_options: {},
       start_time: Date.now(),
-      end_time: Date.now(),
+      end_time: null,
```

Comment:
_nit_: shouldn't that be `undefined` instead? Or maybe we should change `end_time` data type?

Reply:
Yes!